### DISCUSSION TITLE
Migration to v6 issue - CorsMiddleware & IDE Error

### DISCUSSION THREAD

**Discussion by tlap9** [adonisjs/core#4369] [Help] [answered]
Hi guys, I'm doing the migration from v5 to v6, I've been following the docs, application was launched with success but I got a problem when I make a call to my API (first image). Don't know what to do as it is relative to my handler (second image) which is, according to the api-starter-kit, correct.

You can also see my second question on the 2nd image, with the new "app" import, I get an error from my IDE, he asks me to put { app } but this is not working when lauching the app and it is also incorrect following the docs. I've installed all the latest extensions and restarted it but nothing to do. 

I'm really looking forward for your answers as it is not clear at all, for me, how to fix the middleware's class loading from the module.
Thx

![image](https://github.com/adonisjs/core/assets/79936232/dab52ef4-20ae-4629-bb2f-4f2daef30cb4)
![image](https://github.com/adonisjs/core/assets/79936232/e63b61b9-d126-4124-b8bc-3344faf51aec)

**Comment by alaypanov** [comment, marked as answer]
if someone still has that error, it seems cors_provider  is not added to the list of providers during migration.
Just add this import to the adonisrs -> providers array : 
`() => import('@adonisjs/cors/cors_provider')`
